docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -315,34 +319,112 @@ And you will see the following message:
 
 [NOTE]
 ====
-The current access token is used by default to test the service with `Swagger UI` or `GrapghQL UI`. If the provider (other than Keycloak) returns a binary access token then it will be used with `Swagger UI` or `GrapghQL UI` only if this provider has a token introspection endpoint otherwise an `IdToken` which is always in a JWT format will be passed to `Swagger UI` or `GrapghQL UI`. In such cases you can verify with the manual Dev UI test that `401` will always be returned for the current binary access token. Also note that using `IdToken` as a fallback with either of these UIs is only possible with the authorization code flow.
+The current access token is used by default to test the service with Swagger UI or GrapghQL UI. If the provider (other than Keycloak) returns a binary access token, then it is used with Swagger UI or GrapghQL UI only if this provider has a token introspection endpoint; otherwise, an `IdToken`, which is always in a JWT format is passed to Swagger UI or GrapghQL UI. In such cases, you can verify with the manual Dev UI test that `401` is always returned for the current binary access token. Also, note that using `IdToken` as a fallback with either of these user interfaces is only possible with the authorization code flow.
 ====
 
-Some providers such as `Auth0` do not support a standard RP initiated logout so the provider specific logout properties will have to be configured for a logout option be visible. For more information, see xref:security-oidc-code-flow-authentication.adoc#user-initiated-logout[OpenID Connect User-Initiated Logout].
+Some providers, such as `Auth0` do not support a standard RP-initiated logout, so the provider-specific logout properties must be configured for a logout option to be visible. For more information, see xref:security-oidc-code-flow-authentication.adoc#user-initiated-logout[User-initiated logout] section in the "OpenID Connect authorization code flow mechanism for protecting web applications" guide.
 
-Similarly, if you'd like to use a `password` or `client_credentials` grant for Dev UI to acquire the tokens then you may have to configure some extra provider specific properties, for example:
+Similarly, if you want to use a `password` or `client_credentials` grant for Dev UI to acquire the tokens, then you might need to configure some extra provider-specific properties, for example:
 
 [source,properties]
 ----
 quarkus.oidc.devui.grant.type=password
 quarkus.oidc.devui.grant-options.password.audience=http://localhost:8080
 ----
 
-== Non Application Root Path Considerations
+== Dev Services and Dev UI support for other OpenID Connect providers
```

Comment:
@rolfedh FYI, @michalvavrik removed this section as it does not work in new DevUI, so I' not sure why it is visible in your PR

Reply:
Ooops! I must have added it back by accident while resolving the merge conflict. I've removed the section now and checked for any similar errors. I believe this PR is ready for merge now.